src/Components/EligibilityCard/EligibilityCard.tsx

```diff
@@ -1,10 +1,92 @@
 import React from 'react';
-import { View, Text } from 'react-native';
-
-export default function EligibilityCard() {
-  return (
-    <View>
-      <Text>EligibilityCard</Text>
-    </View>
-  );
+import { View, Text, Image } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import styles from './styles';
+import { Case, Eligibility } from '../../types/types';
+import { useRouter } from 'expo-router';
+import Arrow from '../../../assets/arrow.svg';
+import Check from '../../../assets/check.svg';
+import Fileclaim from '../../../assets/file-claim.svg';
+import Optout from '../../../assets/opt-out.svg';
+
+interface EligibilityCardProps {
+  caseData: Case;
+  status: Eligibility;
+}
+
+export default function EligibilityCard({
+  caseData,
+  status,
+}: EligibilityCardProps) {
+  const router = useRouter();
```

Comment:
Let's import `router` instead of `useRouter` for consistency and so that we don't need to declare `router` in every screen.